grpc/interceptors.go

```diff
@@ -22,6 +23,28 @@ func (si *serverInterceptor) intercept(ctx context.Context, req interface{}, inf
 	if info == nil {
 		return nil, berrors.InternalServerError("passed nil *grpc.UnaryServerInfo")
 	}
+
+	// Shave 100 milliseconds off the deadline to ensure that the RPC server times
```

Comment:
typo: I think you're missing an "if" in the sentence "to ensure that the RPC server times out any sub-calls it makes"